front/app/components/PostShowComponents/Comments/Comment/CommentVote/index.tsx

```diff
@@ -177,17 +166,17 @@ const CommentVote = ({
     }
   };
 
-  if (!isNilOrError(comment) && post) {
+  if (!isNilOrError(comment)) {
     let disabled: boolean;
 
-    if (postIsIdea(post)) {
+    if (idea) {
       // Wondering why 'comment_voting_idea' and not 'commenting_idea'?
       // See app/api/ideas/types.ts
       const { enabled, disabled_reason } =
-        post.attributes.action_descriptor.comment_voting_idea;
+        idea.data.attributes.action_descriptor.comment_voting_idea;
       disabled = !enabled && !isFixableByAuthentication(disabled_reason);
     } else {
-      disabled = !!commentVotingPermissionInitiative?.enabled;
+      disabled = !commentVotingPermissionInitiative?.enabled;
```

Comment:
🐛

Reply:
Voting on proposal comments currently not possible when it's enabled (only when it's disabled)